hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -329,10 +341,12 @@ private function up1402_removeUnusedCaseTypes() {
    * Creates/Updates default case types to be shipped
    * with this extension
    *
+   * WARNING: Resets activity types for an assignment, discarding customization
```

Comment:
not sure what do you mean by this , this method only creates a Case Type if the case type is not already exists , so nothing will get discarded here , or am I missing something  ?